package.json

```diff
@@ -54,7 +54,9 @@
   "scripts": {
     "start": "esy b && env $(cat .env | xargs) esy x Server.exe",
     "pesy": "bash -c 'env PESY_MODE=update pesy'",
-    "test": "esy x TestSummarize.exe"
+    "test": "esy x TestSummarize.exe",
+    "docker:prod": "docker-compose up",
+    "docker:dev": "docker-compose -f docker-compose.dev.yml up"
```

Comment:
can you add the scripts to easily run the migration executing the knex thing?

Reply:
sure